### PR TITLE
SW-3249 SW-3254 SW-3265 SW-3346 Numerous Updates to Report Descriptions

### PR DESCRIPTION
**Pull request by charlie-tf**
- info tooltips for many fields
- instructions for Best Months for Observations section
- display=true for fields in view mode
- rm single quotes from annual report section titles

image
image
image

### CODE REVIEW

File: src/strings/csv/en.csv [hunk 1]
```diff
@@ -666,12 +667,12 @@ PROCESSING_AND_DRYING_INFO,"By adding information about processing/drying and sa
...
 PROCESSING_START_DATE,Processing Start Date,
 PROCESSING_STATUS_REQUIRED,Processing Status *,
-PROJECT_IMPACT_REQUIRED, 'Project Impact *',
+PROJECT_IMPACT_REQUIRED,Project Impact *,
 PROJECT_IMPACT_INSTRUCTIONS,"Provide the details of your impact during the reporting period, including key accomplishments and progress toward meeting stated goals and key milestones. In addition to forestry goals, please include jobs created (full-time, part-time, and seasonal), using specific figures disaggregated by gender. (Approximately one page.)",
 PROJECT_INFORMATION,Project Information,
 PROJECT_NOTES,Additional Project Information Notes,
 PROJECT_PHOTOS,Project Photos,
-PROJECT_SUMMARY_REQUIRED, 'Project Summary *',
+PROJECT_SUMMARY_REQUIRED,Project Summary *,
```
Comment (karthikbtf) [inline src/strings/csv/en.csv:675]: just curious.. do these spaces break anything?

File: src/strings/csv/en.csv [hunk 2]
```diff
@@ -732,15 +733,25 @@ REPORT_COULD_NOT_SAVE,Could not save report,
...
 REPORT_NO_SEEDBANKS,No seed banks included in report.,
+REPORT_NURSERY_CAPACITY_INFO,Please list the total number of plants that the nursery can hold using best estimates.,
 REPORT_PROBLEM,Report Problem,
 REPORT_SAVE_AND_CLOSE,Save and Close,
 REPORT_SEEDBANK_BUILD_COMPLETION_DATE,Seed Bank Build Completion Date *,
 REPORT_SEEDBANK_BUILD_START_DATE,Seed Bank Build Start Date *,
 REPORT_SEEDBANK_OPERATION_START_DATE,Seed Bank Operation Start Date *,
+REPORT_SUMMARY_OF_PROGRESS_INFO,Give a brief summary of progress including advancement toward meeting restoration targets as well as social and community benefits in the project.,
+REPORT_TOTAL_NON_TREES_PLANTED_INFO,Please include all other plants planted that are affiliated with the Terraformation project; do not include trees in this figure.,
+REPORT_TOTAL_PAID_WORKERS_INFO,Please list the total number of full-time or part-time paid workers engaged in planting in this project,
```
Comment (karthikbtf) [inline src/strings/csv/en.csv:749]: end with period (and on line 751) or remove them from others?

**Review by karthikbtf — APPROVED** [review]

File: src/strings/csv/en.csv (hunk 1 above)
Comment (charlie-tf) [reply src/strings/csv/en.csv:675]: I'm not sure actually 🤔

File: src/strings/csv/en.csv (hunk 2 above)
Comment (charlie-tf) [reply src/strings/csv/en.csv:749]: Good find. I'll go ahead and change that. Thanks

**Pull request merged**